### ISSUE TITLE
False Positive for network Security Groups Association (Terrascan)

### ISSUE THREAD

**Issue by DAndrewShaw** [github/super-linter#2509]
## Describe the bug

I have Terraform with the VNET and Subnet modules separated.
So, when scanning my Terraform, Terrascan found a false positive saying that I should have network security groups associated with my VNet's Subnet.
I have the NSGs defined with the subnets in that module so I should not be getting this error.
The error is occurring in the VNet module and pointing to the azurerm_virtual_network resource.
When looking at Terrascan repo on GitHub it looks like this was fixed and already merged into main on Feb 2, 2021 (See here for [issue](https://github.com/accurics/terrascan/issues/391))

Not sure if this means the bug is reoccurring in Terrascan or if super-linter is using an older version of Terrascan.

## Expected behavior

No errors found

## Steps to Reproduce

I am using Azure DevOps to run super-linter, command to run:

```
- script: |
    docker pull github/super-linter:latest
    docker run -e RUN_LOCAL=true -e FILTER_REGEX_EXCLUDE=azure-pipelines.yaml -v $(System.DefaultWorkingDirectory):/tmp/lint --name outputDir github/super-linter:latest
  displayName: 'GitHub Super-Linter Code Scan'
```

## Logs

![image](https://user-images.githubusercontent.com/99270276/154565512-14117084-9be6-4110-a52d-039fe0cffc10.png)

**Comment by lindluni** [comment]
It's possible we hadn't yet released a version of Super Linter that included this fix. Can you confirm at the time of running this, that your `latest` tag was pointed at Super-Linter >= v4.9.0

**Comment by DAndrewShaw** [comment]
@lindluni,
I ran this on 2/17. Looking at the command output it references an image from 2/15 which corresponds to the release date of v4.9.0

```
- Image Creation Date:[2022-02-15T14:57:53Z]
- Image Revision:[7f33bdd27cb5eaea9f66f069b90e698b941474b1]
- Image Version:[7f33bdd27cb5eaea9f66f069b90e698b941474b1]
```
